autogen/docs/programming.md.mustache

```diff
@@ -368,14 +367,40 @@ Since many colors are missing from the NetLogo color space, [[approximate-hsb]]
 and [[approximate-rgb]] often can't give you the exact color you ask for, but
 they try to come as close as possible.
 
-Example: you can change any turtle from it's existing NetLogo color to a half
-transparent version of that color using:
+Example:
 
-    set color lput 125 extract-rgb color
+    show extract-rgb approximate-rgb 0 0 255 ;; shows [48 88 161]
+
+This is an approximation, but it is still near NetLogo blue 104.7.
```

Comment:
This example was a little confusing for me.  Maybe split it into two steps?  `let my-color approximate-rgb 0 0 255 ; my-color is 104.7`, then the next line `show extract-rgb my-color`, and in the explanation below add a line about how (48, 88, 161) is pretty far from the starting (0, 0, 255) RGB color (pure blue)?